### ISSUE TITLE
[a11y] 9.1.4.11 Sufficient contrast of graphics and graphic control elements

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10729]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-1-4-11-kontraste-von-grafiken-und-grafischen-bedienelementen-ausreichend

Note: Information-bearing graphics and graphic control elements as well as their states should
states should have a contrast of at least 3:1 to adjacent colors.

### Personal

- [ ] The non-activated checkboxes (frames) do not fulfill the contrast requirements
for graphic control elements, as the current contrast value of 1.3:1
is lower than the required value of 3:1.

- [ ]  The "Search" input field (frame) in the "Enable" area does not meet the contrast requirements for
contrast requirements for graphic control elements, as the current contrast value of
contrast value of 1.2:1 is lower than the required value of 3:1.

### Shares

- [ ] See commentary on Personal.

### Spaces
It is likely that we will have a similar problem with checkboxes on this page as on pages 1 and 2.
as on pages 1 and 2. In the test, the contrasts were OK,
However, this was due to the fact that dark background images were used.

### Deleted Files

- [ ] The "Search" input field (frame) does not fulfill the contrast requirements for graphic
control elements, as the current contrast value of 1.2:1 is lower than the required
required value of 3:1.

### Account

- [ ]  In the "Change password" modal dialog, all input field frames (frames)
do not meet the contrast requirements for graphical control elements, as the current contrast value of
contrast value of 1.2:1 is lower than the required value of 3:1.

- [ ] See comment on Personal.

**Comment by pascalwengerter** [comment]
@tbsbdr this needs an update of the theming variables? Appearently, the input-background color, input-border color and "general" background-color don't fulfill the necessary contrast ratio

**Comment by JammingBen** [comment]
Has been fixed via the linked PRs.
